packages/aragon-wrapper/src/cache/index.js

```diff
@@ -23,44 +19,36 @@ export default class Cache {
       //     os.homedir(), '.aragon', 'cache.json'
       //   )
       // )
-      adapter = new Memory()
-    } else {
-      adapter = new LocalStorage()
     }
-    this.db = low(adapter)
-
-    // Set default cache state
-    this.db.defaults({}).write()
 
     // Set up the changes observable
     this.changes = new Subject()
+
+    // Set up cache DB
+    this.db = localforage.createInstance({
+      driver: [localforage.INDEXEDDB, localforage.LOCALSTORAGE],
+      name: `localforage/${this.prefix}`
+    })
   }
 
   getCacheKeyPath (key) {
     return `${this.prefix}.${key}`
   }
 
-  async set (key, value) {
-    // Some lowdb adapters are synchronous while others are asynchronous so
-    // let's always wrap it in a promise
-    await Promise.resolve(this.db.set(
+  set (key, value) {
+    this.db.setItem(
```

Comment:
I'm not familiar with this part of the codebase so maybe you could help me understand here:
Why are you not `await`ing for the operation to finish (like it was before)? [`setItem`](https://github.com/localForage/localForage#callbacks-vs-promises) returns a `promise`:

```js
set (key, value) {
  await this.db.setItem(
    this.getCacheKeyPath(key),
    value
  )
  this.changes.next({ key, value })
}
```

Reply:
You are right, I need to use await the operation. Daniel point the same thing to me the other day. Thank you